### ISSUE TITLE
Typo in the Copy Constructor of PredicateCombiner

### ISSUE THREAD

**Issue by danoan** [DGtal-team/DGtal#1320]
Instead of using:

myPred1(other.pred1) it should be myPred1(other.myPred1)
myPred2(other.pred2) it should be myPred2(other.myPred2)
myBoolFunctor(other.boolFunctor) it should be myBoolFunctor(other.myBoolFunctor)

base/BasicFunctors.h

```
    PredicateCombiner(  const PredicateCombiner& other )
      : myPred1( other.pred1 ), myPred2( other.pred2 ), myBoolFunctor( other.boolFunctor )
    {
    }
```

**Comment by dcoeurjo** [comment]
hi @danoan, thanks for the feedback

**Comment by dcoeurjo** [comment]
Fixed in #1321
